circle.yml

```diff
@@ -44,5 +44,5 @@ deployment:
     branch: /demo_.*/
     commands:
       - cf login -a https://api.cloud.gov -u gsa-wds_deployer -p $CF_GSA_WDS_PASS -o gsa-wds -s wds-demo
-      - cf zero-downtime-push wds-microsite -f manifest-demo.yml -p _site
```

Comment:
This is good that it's being removed. Less resources for things that don't need to be deployed without downtime like demo branches, leaves more resources for the main sites!